examples/widgets/Widget.js

```diff
@@ -0,0 +1,104 @@
+/**
+ * Generated On: 2016-11-22
+ * Class: Controls
+ * Description: Create a widget.
+ */
+
+function Widget(pName, pElement, options) {
```

Comment:
Sorry for the late answer... You're not too late ! As you mentionned it, it's not possible to do that in `examples`.
In fact, I thought maybe we could do a new `index.html` in `/examples/` for each widget's examples, but it won't solve the problem.
I agree with your solution, I think it's the best we can think about right now. But I don't have any knowledge on `.eslintrc`, so do you agree to help me to do a special `.eslintrc` for the `examples` folder ?